gateway/handler.go

```diff
@@ -580,16 +575,13 @@ func webError(w http.ResponseWriter, err error, defaultCode int) {
 }
 
 func isErrNotFound(err error) bool {
-	return isErrNoLink(err) ||
-		errors.Is(err, routing.ErrNotFound) ||
-		err == routing.ErrNotFound ||
```

Comment:
:thought_balloon:  :+1:  for removing these, it looked like a dead code that could cause hard-to-debug bugs in the futture.